### ISSUE TITLE
Conditional breakpoints can't replace disabled breakpoints

### ISSUE THREAD

**Issue by jasonLaster** [devtools-html/debugger.html#3730]
[Steps to reproduce]:
1.Launch Firefox.
2.Access the https://devtools-html.github.io/debugger-examples/examples/todomvc/ webpage.
3.Open the debugger.
4.Create a breakpoint.
5.Disable the breakpoint.
6.Add a conditional breakpoint on the same line.

[Expected result]:
The disabled breakpoint is replaced with a conditional breakpoint.

[Actual result]:
The disabled breakpoint can't be replaced whith the conditional one.

[Additional Information]
It seems that the conditional breakpoint can only replace the active breakpoint, but the "add conditional breakpoint" is avaliable for the disabled breakpoint as well.

Please observe the attached screencast for further information about this issue.

![Disabled Breakpoint issue.gif](https://shipusercontent.com/9c36971fc42cad3e3a54289eee35cb85/Disabled%20Breakpoint%20issue.gif)

https://bugzilla.mozilla.org/show_bug.cgi?id=1391267

**Comment by jasonLaster** [comment]
Here's a sketch:

```diff
diff --git a/src/actions/breakpoints.js b/src/actions/breakpoints.js
index c26ba5d..ce2b8f3 100644
--- a/src/actions/breakpoints.js
+++ b/src/actions/breakpoints.js
@@ -298,6 +298,10 @@ export function setBreakpointCondition(
       return dispatch(addBreakpoint(location, condition));
     }

+    if (bp.disabled) {
+      await enableBreakpoint(location);
+    }
+
     if (bp.loading) {
       // TODO(jwl): when this function is called, make sure the action
       // creator w
```

**Comment by codehag** [comment]
fixed by #3754
